extras/spiffs/component.mk

```diff
@@ -1,7 +1,30 @@
 # Component makefile for extras/spiffs
 
+# If spiffs is configured as SINGLETON it must be configured in compile time.
+SPIFFS_SINGLETON ?= 1
+
 SPIFFS_BASE_ADDR ?= 0x300000
 SPIFFS_SIZE ?= 0x100000
+SPIFFS_LOG_PAGE_SIZE ?= 256
+SPIFFS_LOG_BLOCK_SIZE ?= 8192
+SPIFFS_ERASE_BLOCK_SIZE ?= 4096
```

Comment:
Then i would recommend hard coding SPIFFS_ERASE_BLOCK_SIZE. If any user
messes with that setting, the file system will inherently fail.

Den 8 aug. 2016 21:10 skrev "sheinz" <notifications@github.com>:

> In extras/spiffs/component.mk
> <https://github.com/SuperHouse/esp-open-rtos/pull/178#discussion_r73935358>
> :
>
> >  SPIFFS_BASE_ADDR ?= 0x300000
> >  SPIFFS_SIZE ?= 0x100000
> > +SPIFFS_LOG_PAGE_SIZE ?= 256
> > +SPIFFS_LOG_BLOCK_SIZE ?= 8192
> > +SPIFFS_ERASE_BLOCK_SIZE ?= 4096
>
> Yes, you are correct, only 4k erase is supported.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperHouse/esp-open-rtos/pull/178/files/5e3ef6de85a1a9e4c11bf4c4e99356b27e75662f#r73935358>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE3NV21TMEu_IKwEqkjtlmqvaGiqv_BQks5qd38xgaJpZM4JfSyC>
> .
>